src/TributeRange.js

```diff
@@ -355,14 +355,22 @@ class TributeRange {
         }
     }
 
-    lastIndexWithLeadingSpace (str, char) {
+    lastIndexWithLeadingSpace (str, trigger) {
         let reversedStr = str.split('').reverse().join('')
         let index = -1
 
         for (let cidx = 0, len = str.length; cidx < len; cidx++) {
             let firstChar = cidx === str.length - 1
             let leadingSpace = /\s/.test(reversedStr[cidx + 1])
-            let match = char === reversedStr[cidx]
+            
+            let match = true;
+            
+            for (let triggerIdx = trigger.length - 1; triggerIdx >= 0; triggerIdx--) {
```

Comment:
Strictly speaking, --triggerIdx would be better (no copy) but something must surely optimise that.

Reply:
I've always assumed these are optimized away.